quoting-service/chart-handler/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Quoting-Service Handler Helm chart for Kubernetes
 name: quoting-service-handler
-version: 15.4.0
-appVersion: v15.7.0
+version: 15.9.0
```

Comment:
```suggestion
version: 16.0.0
```

Reply:
Thought we were leaving this for the release ci bumping?